srcpkgs/ncdu/template

```diff
@@ -10,7 +10,7 @@ license="MIT"
 homepage="http://dev.yorhel.nl/ncdu"
```

Comment:
```suggestion
homepage="https://dev.yorhel.nl/ncdu"
```

Reply:
Done. I also changes the distfiles url to https.